tk_toolchain/cmd_line_tools/tk_build_qt_resources/__init__.py

```diff
@@ -99,54 +99,72 @@ def build_res(
     }
 
 
+def build_absolute_path(path1, path2):
+    dir1 = os.path.dirname(path1)
+    combined_path = os.path.normpath(os.path.join(dir1, path2))
+
+    return combined_path
+
+
+def run_yaml_commands(yaml_file, uic, rcc):
+    with open(yaml_file, "r") as file:
+        yaml = YAML()
+        yaml_commands = yaml.load(file)
```

Comment:
filer is python built-in name. So please do not name variables with that name.

Also, in this case, the variable name is useless:
```suggestion
yaml = YAML()
yaml_commands = yaml.load(open(yaml_file)
```